graphql/quizzes/upsertCodingLanguageQuiz.ts

```diff
@@ -0,0 +1,20 @@
+import { gql } from "@apollo/client";
+
+export const UPSERT_CODING_LANGUAGE_QUIZ_RESPONSE = gql`
+  mutation upsertCodingLanguageQuizResponse(
+    $objects: [coding_language_quiz_insert_input!]!
+  ) {
+    insert_coding_language_quiz(
+      objects: $objects
+      on_conflict: {
```

Comment:
when does a conflict happen for this table?

Reply:
I was using this to handle duplicate rows. But, the new implementation completely removes this.